server/api/dataComplianceApi.js

```diff
@@ -0,0 +1,20 @@
+const dataComplianceApiFactory = client => {
+  const map404ToFalse = error => {
+    if (!error.response) throw error
+    if (!error.response.status) throw error
+    if (error.response.status !== 404) throw error
+    return false
+  }
+
+  const isOffenderRecordRetained = (context, offenderNo) =>
+    client
+      .get(context, `/retention/offenders/${offenderNo}`)
+      .then(response => true)
+      .catch(map404ToFalse)
```

Comment:
With super agent you can determine which errors are successful or not by using `ok`:
e.g: 
`  .ok(res => res.status < 500)`

Reply:
As discussed, we found the client is an abstraction over superagent which doens't expose the ok method.  I've left this as is (at least it's consistent with the other 'api's)